### DISCUSSION TITLE
请教如何连接一个有密码的Redis，只有密码，没有用户名。

### DISCUSSION THREAD

**Discussion by 5high** [IrineSistiana/mosdns#327] [General]
官方的文档里要用redis://username:password@x.x.x.x/0

请问如果我的Redis只设置了密码，没有用户名，这个redis连接应该怎么写呢？

**Comment by crysislinux** [comment]
redis://:[password@x.x.x.x](mailto:password@x.x.x.x)/0

直接不要username应该就行了，别的redis客户端都是这样用

**Reply by 5high** [reply]
感谢回复
